### PR TITLE
user/hyfetch: new package (1.4.11)

### PR DESCRIPTION
**Pull request by cassiofb-dev**
Sadly it does not support chimera yet but users can configure to their liking.

### CODE REVIEW

File: user/hyfetch/template.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = [
+    "bash",
+    "python-setuptools",
+    "python-typing_extensions",
```
Comment (nekopsykose) [inline user/hyfetch/template.py:13]:
```suggestion
```
backport this https://github.com/hykilpikonna/hyfetch/commit/359226c26b065909f348084446ebad4fe0cc0680

File: user/hyfetch/template.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = [
+    "bash",
+    "python-setuptools",
```
Comment (nekopsykose) [inline user/hyfetch/template.py:12]:
```suggestion
```

despite it being mentioned in the setup.py it does not actually depend on this

File: user/hyfetch/template.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+url = "https://github.com/hykilpikonna/hyfetch"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "79cd6706a681fa7c05754b35d9338abf96bebce222c27376a4155e94d6a5350d"
+# no test
+options = ["!check"]
+
+def post_install(self):
+    self.install_license(
+        src="LICENSE.md",
+    )
```
Comment (nekopsykose) [inline user/hyfetch/template.py:27]:
```suggestion
    self.install_license("LICENSE.md"
```

File: user/hyfetch/template.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    self.install_man(
+        src=f"docs/{self.pkgname}.1",
+    )
```
Comment (nekopsykose) [inline user/hyfetch/template.py:31]:
```suggestion
    self.install_man("docs/hyfetch.1")
```

File: user/hyfetch/template.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+    self.install_file(
+        src="README.md",
+        dest=f"usr/share/doc/{self.pkgname}",
+    )
```
Comment (nekopsykose) [inline user/hyfetch/template.py:36]:
```suggestion
```

installing a project readme is not very useful

File: user/hyfetch/template.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+    for shell in ["bash", "zsh"]:
+        self.install_completion(
+            src=f"hyfetch/scripts/autocomplete.{shell}",
+            shell=shell,
+        )
```
Comment (nekopsykose) [inline user/hyfetch/template.py:42]:
```suggestion
        self.install_completion(f"hyfetch/scripts/autocomplete.{shell}", shell)
```

File: user/hyfetch/template.py [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = [
+    "bash",
+    "python-setuptools",
+    "python-typing_extensions",
+]
+pkgdesc = "Neofetch with LGBTQ+ pride flags!"
```
Comment (nekopsykose) [inline user/hyfetch/template.py:15]:
```suggestion
pkgdesc = "Neofetch with pride flags"
```

File: user/hyfetch/template.py (hunk 2 above)
Comment (nekopsykose) [reply user/hyfetch/template.py:12]: actually it is still used in this version specifically

**Comment by cassiofb-dev** [comment]
@nekopsykose good morning, I understand the reason to change but isn't it better to package with the author's original description? Here some packages:

1. https://archlinux.org/packages/extra/any/hyfetch/
2. https://packages.gentoo.org/packages/app-misc/hyfetch
3. https://pkgs.alpinelinux.org/package/edge/testing/x86_64/hyfetch

Thanks for the quick review, I'll fix those lints errors soon on my lunch break and if not replied by then I'll also change the description as suggested.

**Comment by q66** [comment]
https://github.com/hykilpikonna/hyfetch/pull/285 maybe drop in this? seems to work fine (just make a patches/ dir and drop in https://patch-diff.githubusercontent.com/raw/hykilpikonna/hyfetch/pull/285.patch)
Screenshot from 2024-07-11 14-48-43

**Comment by q66** [comment]
as for the pkgdesc i'm fine with either one but i'd drop the exclamation mark regardless since that's in line with punctuation use in any other pkgdesc in the repository

**Pull request merged**